packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -660,6 +663,17 @@ async function generateArmParameterJson(ctx: CoreHookContext) {
       value: envConfig[ResourcePlugins.SimpleAuth][EnvConfigName.SkuName],
     };
   }
+
+  if (envConfig[ResourcePlugins.SimpleAuth]?.[EnvConfigName.Endpoint]) {
```

Comment:
So the previous `env.default.json` does not have a dedicated property for the web app name?